blueprints/component/files/__root__/__templatepath__/__templatename__.hbs

```diff
@@ -0,0 +1 @@
+{{yield}}
```

Comment:
I think the template can be removed since it's no different from the original blueprint.